foundry/https_upload.py

```diff
@@ -0,0 +1,248 @@
+"""
+https_upload.py
```

Comment:
A filename in the docstring for a file is superfluous. I'd recommend deleting.

Reply:
I am glad you brought this up -- this is something I have been doing since I was taught to do so in my early 20s, but it occurs to me now that this pattern probably came from the olden days when there weren't version control systems and it was file chaos. Thanks for challenging this Logan, I haven't thought about it!